bundle/phases/initialize.go

```diff
@@ -39,9 +39,14 @@ func Initialize() bundle.Mutator {
 			mutator.MergePipelineClusters(),
 			mutator.InitializeWorkspaceClient(),
 			mutator.PopulateCurrentUser(),
+
 			mutator.DefineDefaultWorkspaceRoot(),
 			mutator.ExpandWorkspaceRoot(),
 			mutator.DefineDefaultWorkspacePaths(),
+			mutator.PrependWorkspacePrefix(),
+
+			validate.NoWorkspacePrefixUsed(),
```

Comment:
Can you include a comment saying that this must happen before variable interpolation?